compiler/luci/pass/src/FoldShapePass.cpp

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/Pass/FoldShapePass.h"
+
+#include <luci/IR/CircleNodes.h>
+#include <luci/Profile/CircleNodeOrigin.h>
+
+namespace
+{
+
```

Comment:
plz add a comment of text graph like other passes that shows how the graph changes before-after the pass.

Reply:
Ok! I'll update

(Just out of curiosity) Where is the proper place for showing how the graph changes before-after pass?

Some pass commented it at the source of the pass file (_~~Pass.cpp_)
(e.g. `FoldDensifyPass.cpp` / `FoldDequantizePass.cpp`),
and some pass commented it at the test file (_~~Pass.test.cpp_)
(e.g. `FoldCastPass.test.cpp`),
and some pass commented it both of the source file (_~~Pass.cpp_) and the test file (_~~Pass.test.cpp_)
(e.g. `FoldDepthwiseConv2DPass.cpp`, `FoldDepthwiseConv2DPass.test.cpp` / `FoldFullyConnectedPass.cpp`, `FoldFullyConnectedPass.test.cpp`)

Is it the best that showing how the graph changes at the both of the source file and the test file..?

Reply:
What is your understanding by my asking to add a graph?
What is the help of the graph comment in the source do you get?
What is the help of the graph comment in the test do you get?